__tests__/parseGithubLabelEvent.test.ts

```diff
@@ -0,0 +1,549 @@
+import {parseGithubLabelEvent} from '../src/types/githubEvent';
+
+const input = `{
```

Comment:
Did you want to keep this unused example around, so as to be able to look at additional data you might want to extract in the future?